packages/terra-search-field/src/SearchField.jsx

```diff
@@ -60,6 +60,11 @@ const propTypes = {
    */
   onChange: PropTypes.func,
 
+  /**
+   * Function to trigger when user inputs a value. Sends parameters {Event} event and {String} searchText
```

Comment:
Do we need to be more specific with description here like :
>  Function to trigger when user programmatically changes a value OR Function to trigger when value changes programmatically OR something more that explains the purpose this function being added in Search-field.

Reply:
> Found an issue like if we enter the text less than minimum required the message gets displayed saying 'Invalid Text'. this is not clearing out when we immediately add the text programmatically as shown below.
> 
> ![SearchFieldOnInput](https://user-images.githubusercontent.com/45097016/72501493-f6623780-385c-11ea-827e-b68ebe7d7d8a.gif)
> 
> Probably we might have to call handleSearch here
> 
> https://github.com/cerner/terra-core/blob/f1787a51b480734950a587084eaf853e09faabfd/packages/terra-search-field/src/SearchField.jsx#L182
> 
> .

So we have two reasons we don't want handleInput to trigger the search function. 

The first reason is we don't want to double trigger the search when typing value, since onInput and onChange would fire. 

Second reason is that we don't want to give external control over when we search.

cc: @dkasper-was-taken

Reply:
What if we wrote something like `Function to trigger when user inputs a value. Use when programmatically setting a value.` 
Though do we want to mention here that users need to manually dispatch the `input` event if they are programmatically setting?

Reply:
Set a more verbose message here https://github.com/cerner/terra-core/commit/63069a3151a47224cf7d2c3c4ae05354b364304e#diff-105fddc0e7a42eb008cb5c09faad5222R64